script/Arcueid/Day 11/11_00_ARC11_3.txt

```diff
@@ -1381,7 +1381,7 @@ I embraced her a little too forcefully.
 [sha:2c6cfa1493375af72107a2263c1e8cd3092a627c]{
 -- Page 243, Offset 30209.
 -- でも誰が俺を責められよう。
-But who could blame me?
+But who could blame me.
```

Comment:
definitely revert any of these that aren't voiced. Rhetorical or not, questions are followed by question marks in _almost_ all cases.